xml/admin_ceph_gateway.xml

```diff
@@ -2692,349 +2692,279 @@ rgw_override_bucket_index_max_shards = 12
-      zonegroups, and certain configuration options such as storage policies.
+      &ceph; &ogw; supports the notion of realms, which
+      can be a single zone group or multiple zone groups and a globally unique
+      namespace for the realm. Multiple realms provide the ability to support
+      numerous configurations and namespaces.
      </para>
     </listitem>
    </varlistentry>
-   <varlistentry>
-    <term>Realm</term>
-    <listitem>
-     <para>
-      A container for zonegroups. This allows for separation of zonegroups
-      between clusters. It is possible to create multiple realms, making it
-      easier to run completely different configurations in the same cluster.
-     </para>
-    </listitem>
-   </varlistentry>
-   <varlistentry>
-    <term>Period</term>
-    <listitem>
-     <para>
-      A period holds the configuration structure for the current state of the
-      realm. Every period contains a unique ID and an epoch. Every realm has an
-      associated current period, holding the current state of configuration of
-      the zonegroups and storage policies. Any configuration change for a
-      non-master zone will increment the period's epoch. Changing the master
-      zone to a different zone will trigger the following changes:
-     </para>
-     <itemizedlist>
-      <listitem>
-       <para>
-        A new period is generated with a new period ID and epoch of 1.
-       </para>
-      </listitem>
-      <listitem>
-       <para>
-        Realm's current period is updated to point to the newly generated
-        period ID.
-       </para>
-      </listitem>
-      <listitem>
-       <para>
-        Realm's epoch is incremented.
-       </para>
-      </listitem>
-     </itemizedlist>
-    </listitem>
-   </varlistentry>
   </variablelist>
-
   <para>
-   You can configure each &ogw; to participate in a federated architecture,
-   working in an active zone configuration while allowing for writes to
-   non-master zones.
+   You can configure each &ogw; to work in an active-active zone
+   configuration, allowing for writes to non-master zones.
+   The multi-site configuration is stored within a container called a realm.
+   The realm stores zone groups, zones, and a time period with multiple
+   epochs for tracking changes to the configuration. The
+   <literal>ceph-radosgw</literal> daemons handle the synchronization, eliminating the
+   need for a separate synchronization agent. This approach
+   to synchronization allows the &ceph; &ogw; to operate with an
+   active-active configuration instead of active-passive.
   </para>
-
-  <sect2 xml:id="ceph-rgw-fed-term">
-   <title>Terminology</title>
+  <sect2 xml:id="ceph-rgw-multi-req-assump">
+   <title>Requirements and Assumptions</title>
    <para>
-    A description of terms specific to a federated architecture follows:
+     A multi-site configuration requires at least two &ceph; storage clusters;
```

Comment:
The semicolon is wrong here, as there is no list. I suggest replacing it with ", and".

Reply:
good catch :)